src/hooks/useGlobalAppState.ts

```diff
@@ -153,8 +153,6 @@ export const useGlobalAppState = (): void => {
         'useGlobalAppState: subscribing to AppState changes, synchronizing device'
       )
 
-      safePromise(synchronizeDevice)(client)
```

Comment:
Care, by remove this from here, the synchronizeDevice is not called on every app focus anymore (which was the requirement as I remember it)

Reply:
it is, a bit earlier in the file, this one was the first one. For some reason if it's called before the name update, there will be an error from the stack

Reply:
woups, didn't see the other one. So I guess this one was for the first render. So it's ok for me